.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+### Description
+
+For a bug: Describe the bug and list the steps you used when the issue occurred.
+include any files necessary to reproduce this by zipping them and attaching them
+to this issue. For features, describe what you'd like, how it might be used and
+why.
+
+#### Steps to reproduce | Feature Idea (Delete one)
```

Comment:
It's not meant to be a Boolean, just a clear separator. "||" would be the correct and I didn't do that on purpose. Regardless, I accept your suggestion as it's full english, as long as it retains the ALL CAPS for `OR`.